packages/cli/cli/src/cli.ts

```diff
@@ -720,20 +720,32 @@ function addWriteOverridesCommand(cli: Argv<GlobalCliOptions>, cliContext: CliCo
     cli.command(
         "write-overrides",
         "Generate a basic openapi overrides file.",
-        (yargs) =>
+        (yargs) => [
             yargs.option("api", {
                 string: true,
                 description: "Only run the command on the provided API"
             }),
+            yargs.option("include-models", {
```

Comment:
Let's just always default this to true for now

Reply:
Can always delete what we don't like

Reply:
totally